lib/doorkeeper/engine.rb

```diff
@@ -5,7 +5,9 @@ class Engine < Rails::Engine
     end
 
     initializer "doorkeeper.locales" do |app|
-      app.config.i18n.fallbacks = [:en]
+      if app.config.i18n.fallbacks.blank?
+        app.config.i18n.fallbacks = [:en]
+      end
     end
```

Comment:
I believe we want `app.config.i18n.fallbacks += [:en]`. Because doorkeeper can't default to languages it doesn't support we want to _add_ the English option to the ones the application might have defined. This implementation assumes `app.config.i18n.fallbacks` is an array (otherwise we need to initialize it before: `app.config.i18n.fallbacks ||= []`).